src/libs/actions/Policy.js

```diff
@@ -1360,6 +1360,267 @@ function buildOptimisticPolicyRecentlyUsedCategories(policyID, category) {
     return lodashUnion([category], policyRecentlyUsedCategories);
 }
 
+function startCollectBottomUpFlow(iouReport, firstName, lastName) {
```

Comment:
I would call this something more specific to the action the user is taking. Some options: 
- payIOUWithBusinessBankAccount (I think this is most accurate for the action they are taking and everything else that happens is the side-effect of this action) 
- createWorkspaceFromIOUPayment
- createWorkspaceFromBBAPaymentOfIOU
-

Reply:
using the `createWorkspaceFromIOUPayment`